src/curves/bw6_761.rs

```diff
@@ -0,0 +1,329 @@
+use std::ffi::c_uint;
```

Comment:
Oh, this file shouldn't be here, somehow forgot to delete it or/and missed it's autogenerated reintroduction